.github/workflows/coverage.yml

```diff
@@ -1,7 +1,13 @@
 name: Test Coverage
 on:
   pull_request:
+    paths:
+      - "**.go"
```

Comment:
Does the coverage tool already know to skip `_test.go` files?

Reply:
this is really "run `go test`" and "coverage" is sort of a poor name for this workflow. 

My goal is that you run the go tests any time any go file changes.

Reply:
this is really "run `go test`" and "coverage" is sort of a poor name for this workflow. 

My goal is that you run the go tests any time any go file changes.